notebook/services/contents/tests/test_manager.py

```diff
@@ -108,7 +108,7 @@ def test_checkpoint_subdir(self):
         self.assertEqual(cp_dir, os.path.join(root, cpm.checkpoint_dir, cp_name))
         self.assertEqual(cp_subdir, os.path.join(root, subd, cpm.checkpoint_dir, cp_name))
 
-    @dec.skip_win32
+    @dec.skipif(sys.platform == 'win32' and sys.version_info[0] < 3)
```

Comment:
I wondered if that might be the case but I figured if the tests passed on appveyor it would be fine. 

That was based on the assumption that most normal Windows users probably wouldn't be running the tests. In that case it would only affect a Windows developer trying to run the tests without admin privileges where it would fail with a permission error. I think that having some testing on Windows/appveyor is probably worth any hassle that might cause?

OT: In the future it should be possible for developers without admin privileges to create symlinks on Windows 10 - https://blogs.windows.com/buildingapps/2016/12/02/symlinks-windows-10/#oWkhfiQvGP6wte4V.97